scripts/uninstall.sh

```diff
@@ -2,11 +2,9 @@
 set -x
 
 ROOT_DIR=$(dirname $0)/..
-NAMESPACE=${NAMESPACE:-examples}
+NAMESPACE=${NAMESPACE:-object-detection}
 
 helm  del --purge \
 ${NAMESPACE}-videoprocessor
 
-kubectl wait --for=delete --timeout=300s FlinkCluster/video-data-generator -n ${NAMESPACE}
-kubectl wait --for=delete --timeout=300s FlinkCluster/multi-video-grid -n ${NAMESPACE}
```

Comment:
Do not remove these lines.

Reply:
I made a separate scripts folders for you. We can decide on what to do with them later.